drivers/spi/Kconfig.nrfx

```diff
@@ -162,13 +162,29 @@ endif # SPI_2
 
 if SPI_3
```

Comment:
I am confused with the symmetry, here:
Why we have if `SPI_2 `and if `SPI_3 `but `if SPI_1 && !I2C_1`

Reply:
Yes, such checks should be added. They were added to `i2c/Kconfig.nrfx` but not here. I'll fix this.

Reply:
Thanks.